renderer/components/ConsoleOutput.tsx

```diff
@@ -1,41 +1,54 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { Panel } from 'react-bootstrap';
 
-class ConsoleOutput extends React.Component {
-  constructor(props) {
+interface StateProps {
+  toggleConsole: Function;
+  disableScroll: boolean;
+}
+
+interface OwnProps {
+  height: number;
+  output: [];
+  show: boolean;
+}
+
+type Props = StateProps & OwnProps;
+
+export class ConsoleOutput extends React.Component<Props, any> {
```

Comment:
ConsoleOutput has no state, so I made the generic type as `Props, any`. Would it be better to just leave off `any`?

Reply:
Hmm yes, does `React.Component<Props>` work?